tests/core/test_organizer.py

```
@@ -401,3 +404,201 @@ def test_init_vision_processor_failure_returns_none(self, mock_vision_cls: Magic
+        assert text == image == video == audio == cad == []
+        assert len(other) == 1
+        assert other[0].name == "mystery.xyz"
+
+    @pytest.mark.unit
+    @pytest.mark.ci
+    def test_cad_extension(self, tmp_path: Path) -> None:
+        """DWG files are categorized as CAD."""
+        organizer = FileOrganizer.__new__(FileOrganizer)
+        f = tmp_path / "drawing.dwg"
+
+        text, image, video, audio, cad, other = organizer._categorize_files([f])
```

Ruff (0.15.10)

[warning] Unpacked variable `text` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)